### DISCUSSION TITLE
Scan QR, mobile version, multi device

### DISCUSSION THREAD

**Discussion by jleal52** [WhiskeySockets/Baileys#110] [New WhatsApp Features]
Hello,

Does anyone know if it would be possible, in the mobile version, to scan a QR code and add an external device?

This way, Baileys would be the main device, always online without any 14-day disconnections.

Thank you.

**Comment by Riders004** [comment]
> Hello,
> 
> Does anyone know if it would be possible, in the mobile version, to scan a QR code and add an external device?
> 
> This way, Baileys would be the main device, always online without any 14-day disconnections.
> 
> Thank you.

Yes it's always online without getting off upto 15 days if your mobile if off or not connected with internet
